ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/SummaryCards/BySeveritySummaryCard.tsx

```diff
@@ -73,7 +73,12 @@ function BySeveritySummaryCard({
                                     spaceItems={{ default: 'spaceItemsSm' }}
                                     alignItems={{ default: 'alignItemsCenter' }}
                                 >
-                                    {Icon && <Icon color={hasNoResults ? textColor : undefined} />}
+                                    {Icon && (
```

Comment:
Adds a semantic title to the icons so:
1. We can more easily grab onto this element, siblings via Cypress
2. There is an accessible name for an icon that carries meaning more than just being a decorative element